modules/ROOT/pages/_partials/keycloak/configuring-the-service.adoc

```diff
@@ -95,6 +106,7 @@ Depending on the target platform, set the additional value as described in eithe
 
 . Save your changes.
 
-. Create a new user account as described in link:https://www.keycloak.org/docs/3.3/server_admin/topics/users/create-user.html[Creating a New User].
+. Creating a new user account as described in link:{docs-create-user}[Creating a New User].
```

Comment:
@finp - should this follow this syntax? link:url[text^]

Reply:
yes, but given the already large scope of this Pr, lets handle it separately